### PR TITLE
Send voter reg status to Northstar

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
- Added a `updateNorthstarStatus` function to each voter reg import. I know before we've had stuff as traits but I think that makes it a little harder for debugging. Open to suggestions!
- Call this function after we get or create a user.

#### How should this be reviewed?

If you run an import, do these users have `voter_registration_status` set in Northstar?

#### Any background context you want to provide?

Before merge: Update Northstar to have a `voter_registration_status` setter that only allows this field to move up the hierarchy.

#### Relevant tickets

[Card](https://www.pivotaltracker.com/story/show/160103784)

### CODE REVIEW

File: app/Jobs/CreateRockTheVotePostInRogue.php [hunk 1]
```diff
@@ -55,6 +55,8 @@ public function handle(Rogue $rogue)
             }
 
             if (isset($user)) {
+                $this->updateNorthstarStatus($user, $this->translateStatus($this->record['Status'], $this->record['Finish with State']));
+
```
Comment (chloealee) [inline app/Jobs/CreateRockTheVotePostInRogue.php]: @katiecrane I could be totally missing something but I think `updateNorthstarStatus` only takes two params?

**Comment by chloealee** [comment]
> I know before we've had stuff as traits but I think that makes it a little harder for debugging. Open to suggestions!

Totally hear you on this! The only hesitation I have about not using traits is that it seems a little confusing and inconsistent to use traits sometimes and other times to not. So I guess maybe we should just pick which way we want to do and just do that?

File: app/Jobs/CreateRockTheVotePostInRogue.php (hunk 1 above)
Comment (katiecrane) [reply app/Jobs/CreateRockTheVotePostInRogue.php]: @chloealee this is two params! `$user` and `$this->translateStatus($this->record['Status'], $this->record['Finish with State'])`

Comment (chloealee) [reply app/Jobs/CreateRockTheVotePostInRogue.php]: omg my old eyes!

**Comment by chloealee** [comment]
@katiecrane that totally makes sense to me too! I don't have terribly strong feelings on this but if we want to stay consistent and break out the same functions into the traits, maybe we can just add more comments to make it more explicit that we use the function in the trait/or where we don't and hopefully easier to debug?

**Comment by chloealee** [comment]
@katiecrane oooo good call! I didn't realize that it would be in a different trait and I think you're right that that makes sense! Since this would be the only thing in the new trait, maybe it _is_ ok to just leave here but maybe make an `@TODO` comment if there seems to be more overlapping voter reg specific functions to make a new trait? Sorry I keep going back and forth on this!

**Comment by katiecrane** [comment]
@chloealee oooo I like the idea of the `@TODO` for now since it would just be one thing in the trait which feels a lil silly. I've been waffling too! I feel like I do have an opinion but where we land on this is _that_ big of a deal

**Comment by chloealee** [comment]
haha agreed, @katiecrane ! If it feels good to you, I say we just do the `@TODO` for now!

**Pull request merged**